docs/source/basics/multimodal_datasets.rst

```diff
@@ -0,0 +1,285 @@
+.. _multimodal_dataset_usage_label:
+
+===================
+Multimodal Datasets
+===================
+
+Multimodal datasets include both text and image data used to train VLMs. torchtune currently supports multimodal chat datasets.
+
+The primary entry point for fine-tuning with multimodal datasets in torchtune is the :func:`~torchtune.datasets.multimodal.multimodal_chat_dataset`
+builder. This lets you specify a local or Hugging Face dataset that follows the multimodal chat data format
+directly from the config and train your VLM on it.
+
+.. _example_multimodal:
+
+Example multimodal dataset
+--------------------------
+
+Here is an example of a multimodal chat dataset for a visual question-answering task.
```

Comment:
I would add something like: "Notice how <image> is a place holder for the file provided in image_path, so we can know where to insert the image tokens".

I also think that the example should contain more than one image, so people know how to gerenalize it.